docs/src/main/paradox/features/templating.md

```diff
@@ -30,9 +41,9 @@ out: newIndex.html
 
 where `newIndex.html` will be the new name of the generated file. Links leading to this page are automatically updated.
 
-#### layout
+### layout
 
-The `layout` property allows to specify the layout we want to be used for this particular page. The layouts are placed by default in the `target/paradox/theme` folder, but you can create one in `src/main/paradox/_template` folder as a string template file (.st).
+The `layout` property allows to specify the template we want to be used for this particular page. The templates are generated by default in paradox in the `target/paradox/theme` folder, but you can create one in `src/main/paradox/_template` folder as a string template file (.st).
```

Comment:
Suggested change for the first sentence:
The layout of the page is determined by default by a template called `page.st`. Using `layout` property, we can change the layout template to something else for this particular page.